external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceOffset.scala

```diff
@@ -51,4 +60,8 @@ private[kafka010] object KafkaSourceOffset {
   def apply(offsetTuples: (String, Int, Long)*): KafkaSourceOffset = {
     KafkaSourceOffset(offsetTuples.map { case(t, p, o) => (new TopicPartition(t, p), o) }.toMap)
   }
+
+  def apply(serialized: Offset): KafkaSourceOffset = {
```

Comment:
If I get this right, this can be called only on SerializedOffset. In that case, best to type the param as SerializedOffset.